### ISSUE TITLE
Getting an exception System.Net.Sockets.SocketException 

### ISSUE THREAD

**Issue by SanjuJayakumarNair** [microsoft/botframework-components#788]
We are using the Skill Template(Template for botframework skill) but getting socket exception find the below details
Exception type : System.Net.Sockets.SocketException
Exception message: Only one usage of each socket address (protocol/network address/port) is normally permitted.
AspNetCoreEnvironment : Production
In call stack its showing System.Net.Http.HttpRequestException
So could you please help what is the exception and why it is occurring and what to do so we dnt get this exception

**Comment by stevkan** [comment]
For the app service associated with the skill, can you check the configuration's general settings and see if Web Sockets is enabled (set to 'on')?

**Comment by SanjuJayakumarNair** [comment]
Yes checked Web Sockets is enabled (set to 'on') should i do any changes??
